rpc/grpc_example/main.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 var (
-	certificateFile      = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "testnet3", "rpc.cert")
+	certificateFile      = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "rpc.cert")
```

Comment:
Can change back, but I only have a db in "testnet3" and rpc.cert is up a folder by default I think.

Reply:
ah, yes, I had a custom appdata in use there for testnet3

Reply:
Ok then.